demisto_sdk/commands/common/timers.py

```diff
@@ -81,33 +146,58 @@ def report_time_measurements(group_name='Common', time_measurements_dir='time_me
         # sort by the total time
         list.sort(method_states, key=lambda method_stat: float(method_stat[2]), reverse=True)
 
-        write_mesure_to_logger(csv_data=method_states)
-        write_measure_to_file(time_measurements_dir=time_measurements_dir, group_name=group_name, csv_data=method_states)
+        write_measure_to_logger(group_name, csv_data=method_states)
+        write_measure_to_file(time_measurements_dir=time_measurements_dir, name=group_name,
+                              csv_data=method_states)
 
     else:
         logger.debug(f'There is no timers registered for the group {group_name}')
 
 
-def write_mesure_to_logger(csv_data):
+def write_measure_to_logger(name, csv_data, is_packs=False, debug=False):
+    """
+
+    Args:
+        debug: if write to debug
+        name: name of the table
+        csv_data: the data
+        is_packs: if we want to create table for packs (default is table for functions)
```

Comment:
Maybe instead of a boolean (functions/packs) you could introduce a `MEASURE_TYPE` enum?
Maybe in the future we'd want to measure some other type.
The default can be `FUNCTIONS`.
The enum will also have the `name`.